--- ../Turtle/AssociationEvent/AssociationEvent-a.ttl	2021-10-15 16:13:57.976547800 +0300
+++ ../Turtle-simple-context/AssociationEvent/AssociationEvent-a.ttl	2021-10-15 15:31:50.922607100 +0300
@@ -22,6 +21,0 @@
-[ rdf:type         epcis:EPCISDocument ;
-  dcterms:created  "2019-11-01T14:00:00.000+01:00"^^xsd:dateTime ;
-  owl:versionInfo  "2.0" ;
-  epcis:epcisBody  [ epcis:eventList  <ni:///sha-256;025ac144187a8c5e14caf4d1cfa69250a33dc59a5bc42a68d31b1b5e55a3f15a?ver=CBV2.0> ]
-] .
-
@@ -36,0 +31,6 @@
+
+[ rdf:type         epcis:EPCISDocument ;
+  dcterms:created  "2019-11-01T14:00:00.000+01:00"^^xsd:dateTime ;
+  owl:versionInfo  "2.0" ;
+  epcis:epcisBody  [ epcis:eventList  <ni:///sha-256;025ac144187a8c5e14caf4d1cfa69250a33dc59a5bc42a68d31b1b5e55a3f15a?ver=CBV2.0> ]
+] .
