--- ../Turtle/AssociationEvent/AssociationEvent-f.ttl	2021-09-21 12:46:41.000000000 +0100
+++ ../Turtle-simple-context/AssociationEvent/AssociationEvent-f.ttl	2021-09-21 12:46:41.000000000 +0100
@@ -22,6 +21,0 @@
-[ rdf:type         epcis:EPCISDocument ;
-  dcterms:created  "2019-11-06T14:00:00.000+01:00" ;
-  owl:versionInfo  "2.0" ;
-  epcis:epcisBody  [ epcis:eventList  <ni:///sha-256;5f7c472bc4905de27a19b2efc8e4a9c6dc195139669b80b515f12218ff07cf65?ver=CBV2.0> ]
-] .
-
@@ -33 +27 @@
-        epcis:bizTransactionList   [ epcis:bizTransaction      "urn:epcglobal:cbv:bt:4023333000000:54545"^^xsd:anyURI ;
+        epcis:bizTransactionList   [ epcis:bizTransaction      <urn:epcglobal:cbv:bt:4023333000000:54545> ;
@@ -40 +34 @@
-        epcis:destinationList      [ epcis:destination              <urn:epc:id:pgln:4012345.00000> ;
+        epcis:destinationList      [ epcis:sourceOrDestination      <urn:epc:id:pgln:4012345.00000> ;
@@ -44 +38 @@
-        epcis:eventTime            "2019-11-06T14:00:00.000+01:00"^^xsd:dateTimeStamp ;
+        epcis:eventTime            "2019-11-06T14:00:00.000+01:00"^^xsd:dateTime ;
@@ -48,3 +42,3 @@
-        epcis:recordTime           "2019-11-06T14:05:00.000+01:00"^^xsd:dateTimeStamp ;
-        epcis:sensorElementList    [ epcis:sensorMetadata  [ epcis:endTime    "2019-11-06T13:57:00.000+01:00"^^xsd:dateTimeStamp ;
-                                                             epcis:startTime  "2019-11-06T13:55:00.000+01:00"^^xsd:dateTimeStamp
+        epcis:recordTime           "2019-11-06T14:05:00.000+01:00"^^xsd:dateTime ;
+        epcis:sensorElementList    [ epcis:sensorMetadata  [ epcis:endTime    "2019-11-06T13:57:00.000+01:00"^^xsd:dateTime ;
+                                                             epcis:startTime  "2019-11-06T13:55:00.000+01:00"^^xsd:dateTime
@@ -52,0 +47 @@
+                                                             epcis:measurementType  gs1:Humidity ;
@@ -54 +48,0 @@
-                                                             epcis:type      <file://C:/Humidity> ;
@@ -58 +52 @@
-        epcis:sourceList           [ epcis:source                   <urn:epc:id:pgln:4000001.00012> ;
+        epcis:sourceList           [ epcis:sourceOrDestination      <urn:epc:id:pgln:4000001.00012> ;
@@ -60,0 +55,6 @@
+
+[ rdf:type         epcis:EPCISDocument ;
+  dcterms:created  "2019-11-06T14:00:00.000+01:00"^^xsd:dateTime ;
+  owl:versionInfo  "2.0" ;
+  epcis:epcisBody  [ epcis:eventList  <ni:///sha-256;5f7c472bc4905de27a19b2efc8e4a9c6dc195139669b80b515f12218ff07cf65?ver=CBV2.0> ]
+] .
