--- ../Turtle/WithErrorDeclaration/Example_9.6.1-ObjectEvent-with-error-declaration.ttl	2021-09-21 12:46:41.000000000 +0100
+++ ../Turtle-simple-context/WithErrorDeclaration/Example_9.6.1-ObjectEvent-with-error-declaration.ttl	2021-09-21 12:46:41.000000000 +0100
@@ -28 +28 @@
-        epcis:bizTransactionList   [ epcis:bizTransaction      "urn:epcglobal:cbv:bt:0614141073467:1152"^^xsd:anyURI ;
+        epcis:bizTransactionList   [ epcis:bizTransaction      <urn:epcglobal:cbv:bt:0614141073467:1152> ;
@@ -31 +31 @@
-        epcis:bizTransactionList   [ epcis:bizTransaction      "http://transaction.acme.com/po/12345678"^^xsd:anyURI ;
+        epcis:bizTransactionList   [ epcis:bizTransaction      <http://transaction.acme.com/po/12345678> ;
@@ -36 +36 @@
-        epcis:eventTime            "2005-04-04T20:33:31.116-06:00"^^xsd:dateTimeStamp ;
+        epcis:eventTime            "2005-04-04T20:33:31.116-06:00"^^xsd:dateTime ;
@@ -40,6 +39,0 @@
-[ rdf:type         epcis:EPCISDocument ;
-  dcterms:created  "2005-07-11T11:30:47.0Z" ;
-  owl:versionInfo  "2.0" ;
-  epcis:epcisBody  [ epcis:eventList  <ni:///sha-256;aa49daa1fe0b773e0437e546078dc87de9c864d5b9babe84488f31478887fdf3?ver=CBV2.0> , <ni:///sha-256;00e1e6eba3a7cc6125be4793a631f0af50f8322e0ab5f2c0bab994a11cec1d79?ver=CBV2.0> ]
-] .
-
@@ -50 +44 @@
-        epcis:bizTransactionList   [ epcis:bizTransaction      "http://transaction.acme.com/po/12345678"^^xsd:anyURI ;
+        epcis:bizTransactionList   [ epcis:bizTransaction      <http://transaction.acme.com/po/12345678> ;
@@ -56,2 +50,2 @@
-                                     epcis:declarationTime     "2021-02-01T22:46:31.117000+00:00"^^xsd:dateTimeStamp ;
-                                     epcis:reason              <file://C:/incorrect-data>
+                                     epcis:declarationTime     "2021-02-01T22:46:31.117000+00:00"^^xsd:dateTime ;
+                                     epcis:reason              cbv:ER-incorrect-data
@@ -59 +53 @@
-        epcis:eventTime            "2005-04-03T20:33:31.116000-06:00"^^xsd:dateTimeStamp ;
+        epcis:eventTime            "2005-04-03T20:33:31.116000-06:00"^^xsd:dateTime ;
@@ -61,0 +56,6 @@
+
+[ rdf:type         epcis:EPCISDocument ;
+  dcterms:created  "2005-07-11T11:30:47.0Z"^^xsd:dateTime ;
+  owl:versionInfo  "2.0" ;
+  epcis:epcisBody  [ epcis:eventList  <ni:///sha-256;aa49daa1fe0b773e0437e546078dc87de9c864d5b9babe84488f31478887fdf3?ver=CBV2.0> , <ni:///sha-256;00e1e6eba3a7cc6125be4793a631f0af50f8322e0ab5f2c0bab994a11cec1d79?ver=CBV2.0> ]
+] .
