--- ../Turtle//Example_9.6.2-ObjectEvent.ttl	2021-09-21 12:46:41.000000000 +0100
+++ ../Turtle-simple-context//Example_9.6.2-ObjectEvent.ttl	2021-09-21 12:46:41.000000000 +0100
@@ -22,6 +21,0 @@
-[ rdf:type         epcis:EPCISDocument ;
-  dcterms:created  "2005-07-11T11:30:47.0Z" ;
-  owl:versionInfo  "2.0" ;
-  epcis:epcisBody  [ epcis:eventList  <ni:///sha-256;a98f08ae6ac4de3482054314d637c07010b448d3802dccb028a06aafcc6a4b10?ver=CBV2.0> ]
-] .
-
@@ -34,4 +28 @@
-        epcis:destinationList      [ epcis:destination              <urn:epc:id:sgln:0614141.00001.0> ;
-                                     epcis:sourceOrDestinationType  cbv:SDT-owning_party
-                                   ] ;
-        epcis:destinationList      [ epcis:destination              <urn:epc:id:sgln:0614141.00777.0> ;
+        epcis:destinationList      [ epcis:sourceOrDestination      <urn:epc:id:sgln:0614141.00777.0> ;
@@ -39,0 +31,3 @@
+        epcis:destinationList      [ epcis:sourceOrDestination      <urn:epc:id:sgln:0614141.00001.0> ;
+                                     epcis:sourceOrDestinationType  cbv:SDT-owning_party
+                                   ] ;
@@ -41 +35 @@
-        epcis:eventTime            "2013-06-08T14:58:56.591Z"^^xsd:dateTimeStamp ;
+        epcis:eventTime            "2013-06-08T14:58:56.591Z"^^xsd:dateTime ;
@@ -48,4 +42 @@
-        epcis:sourceList           [ epcis:source                   <urn:epc:id:sgln:4012345.00001.0> ;
-                                     epcis:sourceOrDestinationType  cbv:SDT-possessing_party
-                                   ] ;
-        epcis:sourceList           [ epcis:source                   <urn:epc:id:sgln:4012345.00225.0> ;
+        epcis:sourceList           [ epcis:sourceOrDestination      <urn:epc:id:sgln:4012345.00225.0> ;
@@ -52,0 +44,3 @@
+                                   ] ;
+        epcis:sourceList           [ epcis:sourceOrDestination      <urn:epc:id:sgln:4012345.00001.0> ;
+                                     epcis:sourceOrDestinationType  cbv:SDT-possessing_party
@@ -53,0 +48,6 @@
+
+[ rdf:type         epcis:EPCISDocument ;
+  dcterms:created  "2005-07-11T11:30:47.0Z"^^xsd:dateTime ;
+  owl:versionInfo  "2.0" ;
+  epcis:epcisBody  [ epcis:eventList  <ni:///sha-256;a98f08ae6ac4de3482054314d637c07010b448d3802dccb028a06aafcc6a4b10?ver=CBV2.0> ]
+] .
