--- ../Turtle/Example_9.6.1-with-comment.ttl	2021-10-15 16:13:07.299332400 +0300
+++ ../Turtle-simple-context/Example_9.6.1-with-comment.ttl	2021-10-15 15:30:55.235009200 +0300
@@ -49,0 +50,3 @@
+<urn:epcglobal:cbv:bt:0614141073467:1152>
+        epcis:bizTransactionType  cbv:BTT-desadv .
+
@@ -55,3 +57,0 @@
-
-<urn:epcglobal:cbv:bt:0614141073467:1152>
-        epcis:bizTransactionType  cbv:BTT-desadv .
