--- ../Turtle/Example_9.6.3-AggregationEvent.ttl	2021-10-15 16:13:26.428460100 +0300
+++ ../Turtle-simple-context/Example_9.6.3-AggregationEvent.ttl	2021-10-15 15:31:26.533379300 +0300
@@ -22,6 +21,0 @@
-[ rdf:type         epcis:EPCISDocument ;
-  dcterms:created  "2005-07-11T11:30:47.0Z"^^xsd:dateTime ;
-  owl:versionInfo  "2.0" ;
-  epcis:epcisBody  [ epcis:eventList  <ni:///sha-256;87b5f18a69993f0052046d4687dfacdf48f7c988cfabda2819688c86b4066a49?ver=CBV2.0> ]
-] .
-
@@ -35,3 +28,0 @@
-        epcis:childQuantityList    [ epcis:epcClass  <urn:epc:idpat:sgtin:4012345.098765.*> ;
-                                     epcis:quantity  1.0E1
-                                   ] ;
@@ -41,0 +33,3 @@
+        epcis:childQuantityList    [ epcis:epcClass  <urn:epc:idpat:sgtin:4012345.098765.*> ;
+                                     epcis:quantity  1.0E1
+                                   ] ;
@@ -46,0 +41,6 @@
+
+[ rdf:type         epcis:EPCISDocument ;
+  dcterms:created  "2005-07-11T11:30:47.0Z"^^xsd:dateTime ;
+  owl:versionInfo  "2.0" ;
+  epcis:epcisBody  [ epcis:eventList  <ni:///sha-256;87b5f18a69993f0052046d4687dfacdf48f7c988cfabda2819688c86b4066a49?ver=CBV2.0> ]
+] .
