--- ../Turtle/WithSensorData/SensorDataExample11.ttl	2021-09-21 12:46:41.000000000 +0100
+++ ../Turtle-simple-context/WithSensorData/SensorDataExample11.ttl	2021-09-21 12:46:41.000000000 +0100
@@ -29 +29 @@
-        epcis:disposition          <file://C:/needs_replacement> ;
+        epcis:disposition          cbv:Disp-needs_replacement ;
@@ -31 +31 @@
-        epcis:eventTime            "2020-07-03T00:05:00-06:00"^^xsd:dateTimeStamp ;
+        epcis:eventTime            "2020-07-03T00:05:00-06:00"^^xsd:dateTime ;
@@ -34 +34 @@
-        epcis:sensorElementList    [ epcis:sensorReport  [ epcis:type   gs1:MT-DoseEquivalentRate ;
+        epcis:sensorElementList    [ epcis:sensorReport  [ epcis:measurementType  gs1:DoseEquivalentRate ;
@@ -41 +41 @@
-  dcterms:created  "2020-07-03T00:05:00-06:00" ;
+  dcterms:created  "2020-07-03T00:05:00-06:00"^^xsd:dateTime ;
