--- ../Turtle/WithSensorData/SensorDataExample10.ttl	2021-09-21 12:46:41.000000000 +0100
+++ ../Turtle-simple-context/WithSensorData/SensorDataExample10.ttl	2021-09-21 12:46:41.000000000 +0100
@@ -23 +23 @@
-  dcterms:created  "2019-10-07T15:00:00.000+01:00" ;
+  dcterms:created  "2019-10-07T15:00:00.000+01:00"^^xsd:dateTime ;
@@ -31 +31 @@
-        epcis:bizStep              <file://C:/sensor_reporting> ;
+        epcis:bizStep              cbv:BizStep-sensor_reporting ;
@@ -33 +33 @@
-        epcis:eventTime            "2020-05-08T15:00:00+01:00"^^xsd:dateTimeStamp ;
+        epcis:eventTime            "2020-05-08T15:00:00+01:00"^^xsd:dateTime ;
@@ -37,2 +37,2 @@
-                                                           epcis:component  "Z" ;
-                                                           epcis:type       gs1:Speed ;
+                                                           epcis:component        cbv:COMP-Z ;
+                                                           epcis:measurementType  gs1:Speed ;
@@ -43,2 +43,2 @@
-                                                           epcis:component  "Y" ;
-                                                           epcis:type       gs1:Speed ;
+                                                           epcis:component        cbv:COMP-Y ;
+                                                           epcis:measurementType  gs1:Speed ;
@@ -49,2 +49,2 @@
-                                                           epcis:component  "X" ;
-                                                           epcis:type       gs1:Speed ;
+                                                           epcis:component        cbv:COMP-X ;
+                                                           epcis:measurementType  gs1:Speed ;
