--- ../Turtle/WithSensorData/SensorDataExample13.ttl	2021-09-21 12:46:41.000000000 +0100
+++ ../Turtle-simple-context/WithSensorData/SensorDataExample13.ttl	2021-09-21 12:46:41.000000000 +0100
@@ -23 +23 @@
-  dcterms:created  "2021-04-27T15:00:00+01:00" ;
+  dcterms:created  "2021-04-27T15:00:00+01:00"^^xsd:dateTime ;
@@ -31 +31 @@
-        epcis:bizStep              <file://C:/sensor_reporting> ;
+        epcis:bizStep              cbv:BizStep-sensor_reporting ;
@@ -33 +33 @@
-        epcis:eventTime            "2021-04-27T15:00:00+01:00"^^xsd:dateTimeStamp ;
+        epcis:eventTime            "2021-04-27T15:00:00+01:00"^^xsd:dateTime ;
@@ -37 +37 @@
-                                                           epcis:type     gs1:Temperature ;
+                                                           epcis:measurementType  gs1:Temperature ;
@@ -42 +42 @@
-                                                           epcis:type     gs1:Temperature ;
+                                                           epcis:measurementType  gs1:Temperature ;
