--- ../Turtle/WithSensorData/SensorDataExample1.ttl	2021-10-15 16:15:56.676621100 +0300
+++ ../Turtle-simple-context/WithSensorData/SensorDataExample1.ttl	2021-10-15 15:33:50.164820200 +0300
@@ -36 +36 @@
-        epcis:sensorElementList    [ rdf:type              <file://C:/SensorElement> ;
+        epcis:sensorElementList    [ rdf:type              epcis:SensorElement ;
@@ -40 +40 @@
-                                                             epcis:time            "2019-04-02T14:35:00.000+01:00"^^xsd:dateTimeStamp
+                                                             epcis:time            "2019-04-02T14:55:00.000+01:00"^^xsd:dateTimeStamp
@@ -44 +44 @@
-                                                             epcis:value            8.01E2
+                                                             epcis:value            8.02E2
@@ -48 +48 @@
-                                                             epcis:value            1.61E2
+                                                             epcis:value            1.62E2
@@ -56 +56 @@
-                                                             epcis:value            2.61E1
+                                                             epcis:value            2.62E1
@@ -59 +59 @@
-        epcis:sensorElementList    [ rdf:type              <file://C:/SensorElement> ;
+        epcis:sensorElementList    [ rdf:type              epcis:SensorElement ;
@@ -82 +82 @@
-        epcis:sensorElementList    [ rdf:type              <file://C:/SensorElement> ;
+        epcis:sensorElementList    [ rdf:type              epcis:SensorElement ;
@@ -86 +86 @@
-                                                             epcis:time            "2019-04-02T14:55:00.000+01:00"^^xsd:dateTimeStamp
+                                                             epcis:time            "2019-04-02T14:35:00.000+01:00"^^xsd:dateTimeStamp
@@ -90 +90 @@
-                                                             epcis:value            8.02E2
+                                                             epcis:value            8.01E2
@@ -94 +94 @@
-                                                             epcis:value            1.62E2
+                                                             epcis:value            1.61E2
@@ -102 +102 @@
-                                                             epcis:value            2.62E1
+                                                             epcis:value            2.61E1
