--- ../Turtle/WithSensorData/SensorDataExample8.ttl	2021-10-15 16:15:01.088595600 +0300
+++ ../Turtle-simple-context/WithSensorData/SensorDataExample8.ttl	2021-10-15 15:32:56.088801100 +0300
@@ -35,22 +34,0 @@
-        epcis:sensorElementList    [ epcis:sensorMetadata  [ epcis:bizRules              <https://example.org/gdti/4012345000054987> ;
-                                                             epcis:dataProcessingMethod  <https://example.com/gdti/4012345000054987> ;
-                                                             epcis:deviceID              <urn:epc:id:giai:4000001.111> ;
-                                                             epcis:deviceMetadata        <https://id.gs1.org/8004/4000001111> ;
-                                                             epcis:rawData               <https://example.org/8004/401234599999> ;
-                                                             epcis:time                  "2019-07-19T14:00:00.000+01:00"^^xsd:dateTimeStamp
-                                                           ] ;
-                                     epcis:sensorReport    [ epcis:measurementType  gs1:AmountOfSubstancePerUnitVolume ;
-                                                             epcis:microorganism    <https://www.ncbi.nlm.nih.gov/taxonomy/1126011> ;
-                                                             epcis:uom              "C35" ;
-                                                             epcis:value            5.0E-2
-                                                           ] ;
-                                     epcis:sensorReport    [ epcis:chemicalSubstance  <https://identifiers.org/inchikey:CZMRCDWAGMRECN-UGDNZRGBSA-N> ;
-                                                             epcis:measurementType    gs1:AmountOfSubstancePerUnitVolume ;
-                                                             epcis:uom                "C35" ;
-                                                             epcis:value              1.8E-1
-                                                           ] ;
-                                     epcis:sensorReport    [ epcis:measurementType  gs1:AbsoluteHumidity ;
-                                                             epcis:uom              "A93" ;
-                                                             epcis:value            1.21E1
-                                                           ]
-                                   ] ;
@@ -82 +60,2 @@
-                                     epcis:sensorReport  [ epcis:dataProcessingMethod  <https://example.com/gdti/4012345000054987> ;
+                                     epcis:sensorReport  [ epcis:bizRules              <https://example.org/gdti/4012345000054987> ;
+                                                           epcis:dataProcessingMethod  <https://example.com/gdti/4012345000054987> ;
@@ -89,0 +69,22 @@
+                                   ] ;
+        epcis:sensorElementList    [ epcis:sensorMetadata  [ epcis:bizRules              <https://example.org/gdti/4012345000054987> ;
+                                                             epcis:dataProcessingMethod  <https://example.com/gdti/4012345000054987> ;
+                                                             epcis:deviceID              <urn:epc:id:giai:4000001.111> ;
+                                                             epcis:deviceMetadata        <https://id.gs1.org/8004/4000001111> ;
+                                                             epcis:rawData               <https://example.org/8004/401234599999> ;
+                                                             epcis:time                  "2019-07-19T14:00:00.000+01:00"^^xsd:dateTimeStamp
+                                                           ] ;
+                                     epcis:sensorReport    [ epcis:measurementType  gs1:AmountOfSubstancePerUnitVolume ;
+                                                             epcis:microorganism    <https://www.ncbi.nlm.nih.gov/taxonomy/1126011> ;
+                                                             epcis:uom              "C35" ;
+                                                             epcis:value            5.0E-2
+                                                           ] ;
+                                     epcis:sensorReport    [ epcis:chemicalSubstance  <https://identifiers.org/inchikey:CZMRCDWAGMRECN-UGDNZRGBSA-N> ;
+                                                             epcis:measurementType    gs1:AmountOfSubstancePerUnitVolume ;
+                                                             epcis:uom                "C35" ;
+                                                             epcis:value              1.8E-1
+                                                           ] ;
+                                     epcis:sensorReport    [ epcis:measurementType  gs1:AbsoluteHumidity ;
+                                                             epcis:uom              "A93" ;
+                                                             epcis:value            1.21E1
+                                                           ]
