--- ../Turtle/WithSensorData/SensorDataExample12.ttl	2021-10-15 16:15:15.656231400 +0300
+++ ../Turtle-simple-context/WithSensorData/SensorDataExample12.ttl	2021-10-15 15:33:06.382972600 +0300
@@ -39,2 +39 @@
-                                        epcis:ilmd                 [ <cbvmda:lotNumber>       "LOTABC" ;
-                                                                     example:grading          "A" ;
+                                        epcis:ilmd                 [ example:grading          "A" ;
@@ -42 +41,2 @@
-                                                                                       "B-2" ]
+                                                                                       "B-2" ] ;
+                                                                     cbvmda:lotNumber         "LOTABC"
