--- ../Turtle/WithSensorData/SensorDataExample8.ttl	2021-09-21 12:46:41.000000000 +0100
+++ ../Turtle-simple-context/WithSensorData/SensorDataExample8.ttl	2021-09-21 12:46:41.000000000 +0100
@@ -23 +23 @@
-  dcterms:created  "2005-07-11T11:30:47.0Z" ;
+  dcterms:created  "2005-07-11T11:30:47.0Z"^^xsd:dateTime ;
@@ -37 +37 @@
-        epcis:eventTime            "2019-10-07T15:30:00.000+01:00"^^xsd:dateTimeStamp ;
+        epcis:eventTime            "2019-10-07T15:30:00.000+01:00"^^xsd:dateTime ;
@@ -41,2 +41,2 @@
-        epcis:sensorElementList    [ epcis:sensorReport  [ epcis:stringValue  "someSensorOutput" ;
-                                                           epcis:type         examp:someSensorProperty
+        epcis:sensorElementList    [ epcis:sensorReport  [ epcis:measurementType  examp:someSensorProperty ;
+                                                           epcis:stringValue      "someSensorOutput"
@@ -44 +44,2 @@
-                                     epcis:sensorReport  [ epcis:dataProcessingMethod  <https://example.com/gdti/4012345000054987> ;
+                                     epcis:sensorReport  [ epcis:bizRules              <https://example.org/gdti/4012345000054987> ;
+                                                           epcis:dataProcessingMethod  <https://example.com/gdti/4012345000054987> ;
@@ -46,0 +48 @@
+                                                           epcis:measurementType       gs1:Temperature ;
@@ -48,2 +50 @@
-                                                           epcis:time                  "2019-07-19T14:00:00.000+01:00"^^xsd:dateTimeStamp ;
-                                                           epcis:type                  gs1:MT-Temperature ;
+                                                           epcis:time                  "2019-07-19T14:00:00.000+01:00"^^xsd:dateTime ;
@@ -58 +59 @@
-                                                             epcis:time                  "2019-07-19T14:00:00.000+01:00"^^xsd:dateTimeStamp
+                                                             epcis:time                  "2019-07-19T14:00:00.000+01:00"^^xsd:dateTime
@@ -60,2 +61,2 @@
-                                     epcis:sensorReport    [ epcis:microorganism  <https://www.ncbi.nlm.nih.gov/taxonomy/1126011> ;
-                                                             epcis:type           gs1:MT-AmountOfSubstancePerUnitVolume ;
+                                     epcis:sensorReport    [ epcis:measurementType  gs1:AmountOfSubstancePerUnitVolume ;
+                                                             epcis:microorganism    <https://www.ncbi.nlm.nih.gov/taxonomy/1126011> ;
@@ -66 +67 @@
-                                                             epcis:type               gs1:MT-AmountOfSubstancePerUnitVolume ;
+                                                             epcis:measurementType    gs1:AmountOfSubstancePerUnitVolume ;
@@ -70 +71 @@
-                                     epcis:sensorReport    [ epcis:type   <file://C:/Humidity> ;
+                                     epcis:sensorReport    [ epcis:measurementType  gs1:Humidity ;
@@ -80,2 +81,2 @@
-                                                                epcis:endTime              "2019-04-02T14:59:59.999+01:00"^^xsd:dateTimeStamp ;
-                                                                epcis:startTime            "2019-04-01T15:00:00.000+01:00"^^xsd:dateTimeStamp
+                                                                epcis:endTime              "2019-04-02T14:59:59.999+01:00"^^xsd:dateTime ;
+                                                                epcis:startTime            "2019-04-01T15:00:00.000+01:00"^^xsd:dateTime
@@ -83,2 +84,2 @@
-                                     epcis:sensorReport       [ epcis:stringValue  "someSensorOutput" ;
-                                                                epcis:type         examp:someSensorProperty
+                                     epcis:sensorReport       [ epcis:measurementType  examp:someSensorProperty ;
+                                                                epcis:stringValue      "someSensorOutput"
@@ -88,0 +90 @@
+                                                                epcis:measurementType  gs1:Temperature ;
@@ -93 +94,0 @@
-                                                                epcis:type       gs1:MT-Temperature ;
