--- ../Turtle/WithSensorData/SensorDataExample9.ttl	2021-09-21 12:46:41.000000000 +0100
+++ ../Turtle-simple-context/WithSensorData/SensorDataExample9.ttl	2021-09-21 12:46:41.000000000 +0100
@@ -23 +23 @@
-  dcterms:created  "2019-10-07T15:00:00.000+01:00" ;
+  dcterms:created  "2019-10-07T15:00:00.000+01:00"^^xsd:dateTime ;
@@ -27 +27 @@
-                                        epcis:bizStep              <file://C:/sensor_reporting> ;
+                                        epcis:bizStep              cbv:BizStep-sensor_reporting ;
@@ -29 +29 @@
-                                        epcis:eventTime            "2020-05-07T15:00:00.000Z"^^xsd:dateTimeStamp ;
+                                        epcis:eventTime            "2020-05-07T15:00:00.000Z"^^xsd:dateTime ;
@@ -33 +33 @@
-                                                                                           epcis:type          <file://C:/ALARM_CONDITION>
+                                                                                           epcis:measurementType  gs1:ALARM_CONDITION
@@ -35,2 +35,2 @@
-                                                                     epcis:sensorReport  [ epcis:type      <file://C:/ERROR_CONDITION> ;
-                                                                                           epcis:uriValue  "https://example.com/ErrorCode-A827"^^xsd:anyURI
+                                                                     epcis:sensorReport  [ epcis:measurementType  gs1:ERROR_CONDITION ;
+                                                                                           epcis:uriValue         <https://example.com/ErrorCode-A827>
