--- ../Turtle/WithSensorData/SensorDataExample3.ttl	2021-09-21 12:46:41.000000000 +0100
+++ ../Turtle-simple-context/WithSensorData/SensorDataExample3.ttl	2021-09-21 12:46:41.000000000 +0100
@@ -21,0 +22,6 @@
+[ rdf:type         epcis:EPCISDocument ;
+  dcterms:created  "2005-07-11T11:30:47.0Z"^^xsd:dateTime ;
+  owl:versionInfo  "2.0" ;
+  epcis:epcisBody  [ epcis:eventList  <ni:///sha-256;b507b2720c96cd99855eabb101e8617d48622eff4ceaad0a416b3c10dfd316ee?ver=CBV2.0> ]
+] .
+
@@ -26 +32 @@
-        epcis:eventTime            "2019-04-02T15:00:00.000+01:00"^^xsd:dateTimeStamp ;
+        epcis:eventTime            "2019-04-02T15:00:00.000+01:00"^^xsd:dateTime ;
@@ -29,2 +35,2 @@
-        epcis:sensorElementList    [ epcis:sensorMetadata  [ epcis:endTime    "2019-04-02T14:59:59.999+01:00"^^xsd:dateTimeStamp ;
-                                                             epcis:startTime  "2019-04-01T15:00:00.000+01:00"^^xsd:dateTimeStamp
+        epcis:sensorElementList    [ epcis:sensorMetadata  [ epcis:endTime    "2019-04-02T14:59:59.999+01:00"^^xsd:dateTime ;
+                                                             epcis:startTime  "2019-04-01T15:00:00.000+01:00"^^xsd:dateTime
@@ -32,0 +39 @@
+                                                             epcis:measurementType  gs1:Humidity ;
@@ -34 +40,0 @@
-                                                             epcis:type      <file://C:/Humidity> ;
@@ -37,0 +44 @@
+                                                             epcis:measurementType  gs1:Temperature ;
@@ -39 +45,0 @@
-                                                             epcis:type      gs1:MT-Temperature ;
@@ -43,6 +48,0 @@
-
-[ rdf:type         epcis:EPCISDocument ;
-  dcterms:created  "2005-07-11T11:30:47.0Z" ;
-  owl:versionInfo  "2.0" ;
-  epcis:epcisBody  [ epcis:eventList  <ni:///sha-256;b507b2720c96cd99855eabb101e8617d48622eff4ceaad0a416b3c10dfd316ee?ver=CBV2.0> ]
-] .
