--- ../Turtle/WithSensorData/SensorDataExample15.ttl	2021-10-15 16:14:51.794808700 +0300
+++ ../Turtle-simple-context/WithSensorData/SensorDataExample15.ttl	2021-10-15 15:32:47.269293500 +0300
@@ -34 +34 @@
-                                                                     epcis:sensorReport    [ epcis:component        cbv:Comp-easting ;
+                                                                     epcis:sensorReport    [ epcis:component        cbv:COMP-easting ;
@@ -41 +41 @@
-                                                                     epcis:sensorReport    [ epcis:component        cbv:Comp-northing ;
+                                                                     epcis:sensorReport    [ epcis:component        cbv:COMP-northing ;
