--- ../Turtle/WithSensorData/SensorDataExample9.ttl	2021-10-15 16:15:34.870530400 +0300
+++ ../Turtle-simple-context/WithSensorData/SensorDataExample9.ttl	2021-10-15 15:33:28.496682000 +0300
@@ -33 +33 @@
-                                                                                           epcis:exception        gs1:ALARM_CONDITION ;
+                                                                                           epcis:exception        <https://ns.gs1.org/voc/ALARM_CONDITION> ;
@@ -36 +36 @@
-                                                                     epcis:sensorReport  [ epcis:exception        gs1:ERROR_CONDITION ;
+                                                                     epcis:sensorReport  [ epcis:exception        <https://ns.gs1.org/voc/ERROR_CONDITION> ;
