--- ../Turtle/WithDigitalLinkID/Example_9.6.2-ObjectEventWithDigitalLink.ttl	2021-10-15 16:14:11.494527800 +0300
+++ ../Turtle-simple-context/WithDigitalLinkID/Example_9.6.2-ObjectEventWithDigitalLink.ttl	2021-10-15 15:32:02.363960100 +0300
@@ -28,3 +27,0 @@
-        epcis:destinationList      [ epcis:sourceOrDestination      <urn:epc:id:sgln:0614141.00001.0> ;
-                                     epcis:sourceOrDestinationType  cbv:SDT-owning_party
-                                   ] ;
@@ -32,0 +30,3 @@
+                                   ] ;
+        epcis:destinationList      [ epcis:sourceOrDestination      <urn:epc:id:sgln:0614141.00001.0> ;
+                                     epcis:sourceOrDestinationType  cbv:SDT-owning_party
