--- ../Turtle/WithDigitalLinkID/Example_9.6.4-TransformationEventWithDigitalLink.ttl	2021-10-15 16:14:05.254055400 +0300
+++ ../Turtle-simple-context/WithDigitalLinkID/Example_9.6.4-TransformationEventWithDigitalLink.ttl	2021-10-15 15:31:58.714616800 +0300
@@ -21,0 +22,6 @@
+[ rdf:type         epcis:EPCISDocument ;
+  dcterms:created  "2013-06-04T14:59:02.099+02:00"^^xsd:dateTime ;
+  owl:versionInfo  "2.0" ;
+  epcis:epcisBody  [ epcis:eventList  <ni:///sha-256;4f143d1adf7b2950a34f5e82a240ce5280530b06a9f3c2b9cfe49f5ca5001815?ver=CBV2.0> ]
+] .
+
@@ -33,2 +39,2 @@
-        epcis:inputQuantityList    [ epcis:epcClass  <https://id.gs1.org/01/00614141777778/10/987> ;
-                                     epcis:quantity  3.0E1
+        epcis:inputQuantityList    [ epcis:epcClass  <https://id.gs1.org/01/04012345666663> ;
+                                     epcis:quantity  2.2E2
@@ -40,2 +46,2 @@
-        epcis:inputQuantityList    [ epcis:epcClass  <https://id.gs1.org/01/04012345666663> ;
-                                     epcis:quantity  2.2E2
+        epcis:inputQuantityList    [ epcis:epcClass  <https://id.gs1.org/01/00614141777778/10/987> ;
+                                     epcis:quantity  3.0E1
@@ -45,6 +50,0 @@
-
-[ rdf:type         epcis:EPCISDocument ;
-  dcterms:created  "2013-06-04T14:59:02.099+02:00"^^xsd:dateTime ;
-  owl:versionInfo  "2.0" ;
-  epcis:epcisBody  [ epcis:eventList  <ni:///sha-256;4f143d1adf7b2950a34f5e82a240ce5280530b06a9f3c2b9cfe49f5ca5001815?ver=CBV2.0> ]
-] .
